modules/openapi-generator/src/main/resources/typescript-axios/api.mustache

```diff
@@ -0,0 +1,368 @@
+/// <reference path="./custom.d.ts" />
```

Comment:
I can't find differences to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/typescript-fetch/api.mustache

Reply:
I believe @nicokoenig has made some updates to use [axios](https://github.com/axios/axios). 

@nicokoenig have you committed those changes to this PR?

Reply:
Not yet, changes to come. I made the PR on behalf of @wing328 so that he could check for initial mistakes 👍